auto_round/__main__.py

```diff
@@ -66,6 +66,23 @@ def __init__(self, *args, **kwargs):
             help="The batch size for tuning/calibration."
             "Larger batch sizes may improve stability but require more memory.",
         )
+        self.add_argument("--avg_bits", default=None, type=float, help="for auto scheme, number of avg weight bits")
+        self.add_argument(
+            "--options", default=None, type=str, help="for auto scheme, options for auto scheme, e.g. 'W4A16,W8A16'"
+        )
+        self.add_argument(
+            "--ignore_scale_zp_bits",
+            action="store_true",
+            help="for auto scheme whether ignore scale zp bits calculation ",
+        )
+        self.add_argument("--bits", default=None, type=int, help="number of weight bits")
+        self.add_argument("--group_size", default=None, type=int, help="group size")
+        self.add_argument("--asym", action="store_true", help="whether to use asym quantization")
+        self.add_argument("--data_type", "--dtype", default=None, help="data type for tuning, 'int', 'mx_fp' and etc")
```

Comment:
avg_bits and options should be enough, so data_type is optional under tuning, right?

Reply:
data_type is retained for backward compatibility, as it was previously used in place of scheme. Currently, all keys from scheme are also included in the arguments to make it easier for users to customize or modify the quantization scheme.
For example, if a predefined w2g32 scheme is not available, users can manually specify:
~~~bash
--scheme W2A16 --group_size 32
~~~
in which W2A16 is with group_size 128 

The same applies when using data_type.